packages/story-editor/src/components/library/panes/text/stylePresets/stylePresets.js

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  Headline,
+  THEME_CONSTANTS,
+  Text,
+  BUTTON_TYPES,
+  BUTTON_SIZES,
+  BUTTON_VARIANTS,
+  Icons,
+  Button,
+  PLACEMENT,
+  Popup,
+} from '@googleforcreators/design-system';
+import { __ } from '@googleforcreators/i18n';
+import styled from 'styled-components';
+import { useCallback, useRef, useState, memo } from '@googleforcreators/react';
+import { useFeature } from 'flagged';
+
+/**
+ * Internal dependencies
+ */
+import { useStory, useConfig } from '../../../../../app';
+import { PRESET_TYPES } from '../../../../../constants';
+import useAddPreset from '../../../../../utils/useAddPreset';
+import { DEFAULT_PRESET } from '../textPresets';
+import { focusStyle } from '../../../../panels/shared';
+import StyleGroup from '../../../../styleManager/styleGroup';
+import StyleManager from '../../../../styleManager';
+import useLibrary from '../../../useLibrary';
+import { areAllType } from '../../../../../utils/presetUtils';
+import useApplyStyle from '../../../../styleManager/useApplyStyle';
+import updateProperties from '../../../../inspector/design/updateProperties';
+import getUpdatedSizeAndPosition from '../../../../../utils/getUpdatedSizeAndPosition';
+import InsertionOverlay from '../../shared/insertionOverlay';
+import { Container } from '../../../common/section';
+
+const SectionContainer = styled(Container)`
+  padding: 16px 0;
+  margin-bottom: -24px;
+  border-top: 1px solid ${({ theme }) => theme.colors.divider.tertiary};
+  border-bottom: 1px solid ${({ theme }) => theme.colors.divider.tertiary};
+`;
+
+const TitleBar = styled.div`
+  display: flex;
+  padding-bottom: 16px;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const StylesWrapper = styled.div``;
+
+const StyledButton = styled(Button)`
+  margin-left: auto;
+`;
+
+const StyledMoreButton = styled(Button)`
+  ${focusStyle};
+  margin-top: 12px;
+  padding: 0 16px;
+  justify-content: center;
+  align-self: center;
+  width: 100%;
+  svg {
+    transform: rotate(-90deg);
+    height: 32px;
+    width: 32px;
+    margin-left: -4px;
+  }
+`;
+
+const NoStylesWrapper = styled.div`
```

Comment:
There are some parts of this file that are shared with the Design Panel `stylePresets`, e.g. `NoStylesWrapper`. Not sure if there's a good single place where to keep those instead. Ideas?

Reply:
Hm, you did create that new style manager folder. Perhaps some related shared components could go there too?